backupctl/backupctl.py

```diff
@@ -142,7 +142,7 @@ def config():
     :rtype: `configparser.ConfigParser`
     """
     cfg = configparser.ConfigParser()
-    cfg.read(os.path.join(os.sep, 'etc', 'backupctl.db'))
+    cfg.read(os.path.join(os.sep, 'etc', 'backupctl.ini'))
```

Comment:
BTW no need to use `os.sep` here - it's not going to run on Windows anyhow. Just hardcode `/etc` instead.

Reply:
I used ``os.sep`` everytime, so if, then I should change it everywhere is used ``os.sep`` and not only here.